solutions/platinum/cses-1742.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: cses-1742
+source: CSES
+title: Robot Path
+author: Mihnea Brebenel
+---
+
+## Explanation
+As you have already guessed this is a sweep line problem.
+
+Before we start talking about the actual solution, we need to make some observations about the problem's constraints. The coordinates can get easily big enough
+to cause us some issues. To solve this use [coordinate compresstion](/silver/sorting-custom?lang=cpp).
```

Comment:
```suggestion
Before we start talking about the actual solution, we need to make some observations about the problem's constraints. The coordinates can get easily big enough
to cause us some issues. To solve this use [coordinate compresstion](/silver/sorting-custom?lang=cpp).
```

this is really unclear - could you mention how it exceeds and what issues it would cause.

Reply:
Done.